data/paradise world jobs.txt

```diff
@@ -403,3 +403,92 @@ mission "Pleasure cruise security"
+	passengers 20 100 .6
+	to offer
+		random < 15
+		"passenger space" > 50
+	source
+		attributes "mining" "fishing" "textiles" "factory" "farming" "oil"
+	destination
+		distance 2 18
+		attributes "paradise"
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		payment 10000
+		dialog "The passengers shuffle unhappily out of the ship under the watchful eye of a local security force as they prepare to repay their debts to society. A clerk on <origin> checks their condition as they're led off. He seems mostly pleased and hands you your payment of <payment>."
+
+
+mission "Debtors to <planet>"
+	job
+	repeat
+	description `These <bunks> poor souls have bankrupted themselves attempting to live beyond their means on <origin>. Due to their general incompetence, and media scrutiny on the issue, the government will pay you <payment> to assist in deporting them to somewhere they can make something useful of themselves.`
+	passengers 20 100 .6
+	to offer
+		random < 15
+		"passenger space" > 50
+	source
+		attributes "paradise"
+	destination
+		distance 2 30
+		attributes "mining" "fishing" "forest" "textiles" "factory" "farming" "oil" "frontier" "dirt belt" "south" "rim"
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		payment 10000
+		dialog "The passengers shuffle unhappily out of the ship under the watchful eye of immigration officials. A grinning job service provider is on hand to pay you the <payment> you are owed."
+
+
+# Luxury Accomodation bulk missions
+mission "Wealthy Retirees to <planet>"
+	job
+	repeat
+	description "This coterie of <bunks> wealthy retirees is willing to pay <payment> to be transported in style to <destination>, along with <tons> of their most treasured possessions."
+	passengers 7 12 .6
+	cargo "treasured possessions" 7 12 .2
+	to offer
+		random < 15
+		"passenger space" > 30
+		"cargo space" > 120
+	on offer
+		require "Luxury Accommodations"
+	source
+		attributes "frontier" "dirt belt" "south" "farming" "mining" "rim" "forest" "urban"
+	destination
+		distance 3 16
+		attributes "paradise"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 10000 200
+		dialog "You bid goodbye to the retirees, who were somewhat prone to spilling drinks or irritating you whilst attempting to do work. They were otherwise extremely polite and full of highly amusing, though often morally questionable, anecdotes. Once the last of their things have been safely unloaded your accounts receive the payment of <payment> from their travel broker."
+
+
+mission "Wilderness Retreat Cohort to <planet>"
+	job
+	repeat
+	description "Bring <bunks> pseudo-celebrities and wealthy tourists to the wild and untamed world of <destination> for a 'Wilderness Retreat'. Payment is <payment>."
```

Comment:
Pseudo-celebrities? Is that what they call them kids on the tick tocks these days? 
```suggestion
	description `Bring <bunks> pseudo-celebrities and wealthy tourists to the wild and untamed world of <destination> for a "Wilderness Retreat." Payment is <payment>.`
```